version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "0.1-SNAPSHOT"
+ThisBuild / version := "0.1.0"
```

Comment:
**question:** Should we bump the version here already? Or rather wait until this is merged and autimatically released by CI?

Reply:
Good point! I have changed it to 0.1.0-SHAPSHOT and check:

```
Release version [0.1.0] :
Next version [0.1.1-SNAPSHOT]
```

So fix was commited!